packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -19,6 +19,8 @@ export interface ICalendarMonthProps extends React.Props<CalendarMonth> {
   onNavigateDate: (date: Date, focusOnNavigatedDay: boolean) => void;
   today?: Date;
   highlightCurrentMonth: boolean;
+  showMonthPickerAsOverlay?: boolean;
+  onSelectSwitchCalendar: (focus: boolean) => void;
```

Comment:
Same comments as for DayPicker, rather just have this component know whether the header is clickable